modules/serverless-subscribing-a-function-to-cloudevents.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: PROCEDURE
+[id="serverless-subscribing-a-function-to-cloudevents_{context}"]
+= Subscribing a function to CloudEvents
+
+The `subscribe` command connects the function to a set of events, matching a series of filters for `CloudEvent` metadata and a Knative Broker as the source of events, from where they are consumed.
+
+.Prerequisites
+
+* You have installed Knative Eventing on the cluster.
+* You have configured a Knative Broker.
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+. Subscribe the function to events for a given broker:
```

Comment:
Consider the following list for [Procedure step that introduces a command](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command), though it looks like the rest of the book doesn't really adhere either. 🤷 So maybe something for a future sweep.